xm-commons-timeline/src/main/java/com/icthh/xm/commons/timeline/config/TimelineInterceptor.java

```diff
@@ -26,27 +31,31 @@ public class TimelineInterceptor extends HandlerInterceptorAdapter {
     private static final String HEADER_TENANT = "x-tenant";
     private static final String AUTH_TENANT_KEY = "tenant";
     private static final String AUTH_USER_KEY = "user_key";
+    private static final String TYPE_KEY = "typeKey";
 
     private final AntPathMatcher matcher = new AntPathMatcher();
 
     private final TimelineEventProducer eventProducer;
     private final List<String> ignoredPatterns;
     private final List<String> ignoredHttpMethods;
+    private final List<String> ignoredTypeKeys;
 
     public TimelineInterceptor(
         TimelineEventProducer eventProducer,
         @Value("${application.tenant-ignored-path-list:true}") List<String> ignoredPatterns,
-        @Value("${application.timeline-ignored-http-methods:#{T(java.util.Collections).emptyList()}}") List<String> ignoredHttpMethods
-    ) {
+        @Value("${application.timeline-ignored-http-methods:#{T(java.util.Collections).emptyList()}}") List<String> ignoredHttpMethods,
+        @Value("${application.timeline-ignored-typekeys:#{T(java.util.Collections).emptyList()}}") List<String> ignoredTypeKeys
```

Comment:
For me better will be use kebab case as it was before: timeline-ignored-type-keys, what do you think?